webpack.config.client.js

```diff
@@ -80,6 +80,7 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
       new webpack.DefinePlugin({
         'process.env': getClientEnvVars(DOT_ENV_CONFIG),
       }),
+      new webpack.HashedModuleIdsPlugin(),
```

Comment:
```suggestion
      /**
       * Needed to prevent bundle hashes changing when the order they're imported is changed.
       * See https://webpack.js.org/guides/caching/#module-identifiers
       */
      new webpack.HashedModuleIdsPlugin(),
```